cmd/client.go

```diff
@@ -149,20 +160,13 @@ func (cpc *CloudproberClient) RegisterAndAddProbe(extensionNumber int, probePb *
 // - error:
 //	   - Code: 3, InvalidArgument: probeName is an empty string
 //	   - Code: 5,  NotFound: cannot find a probe matching this probe name
-//	   - ClientNotInitialised: the gRPC client of CloudproberClient is not registered.
-//	     -> Solution: Call the InitClient(rpcServer string) method.
-func (cpc *CloudproberClient) RemoveProbe(probeName string) error {
-	cpc.clientMux.Lock()
-	defer cpc.clientMux.Unlock()
-
-	if cpc.client == nil {
-		err := fmt.Errorf("ClientNotInitialised: cannot register and add a probe when client is not initialised")
-		return err
-	}
+func (client *CloudproberClient) RemoveProbe(probeName string) error {
+	client.clientMux.Lock()
+	defer client.clientMux.Unlock()
 
 	probeToRemove := &probeName // Need to use a string pointer for RemoveProbeRequest{}
```

Comment:
You can directly use `&probeName` below.

Reply:
Amended.